network/src/connectivity_manager/test.rs

```diff
@@ -1138,23 +1130,23 @@ fn multiple_addrs_shrinking() {
 
         // Send addresses of other peer.
         info!("Sending address of other peer");
-        conn_mgr_reqs_tx
-            .send(ConnectivityRequest::UpdateAddresses(
-                DiscoverySource::OnChain,
-                [(
-                    other_peer_id,
-                    vec![
-                        other_addr_1.clone(),
-                        other_addr_2.clone(),
-                        other_addr_3.clone(),
-                    ],
-                )]
-                .iter()
-                .cloned()
-                .collect(),
-            ))
-            .await
-            .unwrap();
+        send_update_addresses(
+            &mut conn_mgr_reqs_tx,
+            DiscoverySource::OnChain,
+            [(
+                other_peer_id,
+                vec![
+                    other_addr_1.clone(),
+                    other_addr_2.clone(),
+                    other_addr_3.clone(),
+                ],
+            )]
+            .iter()
+            .cloned()
+            .collect(),
```

Comment:
man, we really need like a hashmap! macro or something...

Reply:
I was considering making one for these tests